### DISCUSSION TITLE
Looking for simple configuration

### DISCUSSION THREAD

**Discussion by Bohan-Song** [RavenSystem/esp-homekit-devices#2065] [General]
I like your project very much and use it often, but I have no programming foundation;
my English is very poor, json configuration is very difficult for me;
I want to find a method similar to using graphical configuration json, I tried  **https: //glumb.github.io/haa-configurator/**
but it is no longer updated, is there a simpler way to configure json, thank you!

**Comment by Bene2103** [comment]
You can have a look into my programm, its still in developement and right now in an early stage.

https://github.com/Bene2103/HaaConfigurator/
